test/paragraph-cases/dunderproto.html

```diff
@@ -0,0 +1,2 @@
+<p>__proto__ should render verbatim, likewise Object.prototype.__proto__
+</p>
```

Comment:
I like keeping it as it makes the output HTML vaguely formatted like the input document :-P Up to you, though, not too hard to remove (although not as simple as a .trim() :-P)